recipe/build.sh

```diff
@@ -10,3 +9,14 @@ bash configure  --prefix=$PREFIX $OPTS
 make
 make check
 make install
+
+if [ "$(uname)" == "Darwin" ]; then
+    mv nextstep/Emacs.app $PREFIX/Emacs.app
+    mkdir -p $PREFIX/bin
+    ln -s $PREFIX/Emacs.app/Contents/MacOS/Emacs $PREFIX/bin/emacs-25.1
+    ln -s $PREFIX/bin/emacs-25.1 $PREFIX/bin/emacs
+    ln -s $PREFIX/Emacs.app/Contents/MacOS/Emacs/bin/ctags $PREFIX/bin/ctags
+    ln -s $PREFIX/Emacs.app/Contents/MacOS/Emacs/bin/ebrowse $PREFIX/bin/ebrowse
+    ln -s $PREFIX/Emacs.app/Contents/MacOS/Emacs/bin/emacsclient $PREFIX/bin/emacsclient
+    ln -s $PREFIX/Emacs.app/Contents/MacOS/Emacs/bin/etags $PREFIX/bin/etags
```

Comment:
Right, I'm just wondering if this is somehow causing the failure. Not that doing this should be a problem. Definitely looks like a `conda-build` bug. This in particular is suspect.

```
NotADirectoryError: [Errno 20] Not a directory: '/Users/travis/miniconda3/envs/_build_placehold_placehold_placehold_placehold_pla/bin/*'
```